features/step_definitions/user_steps.rb

```diff
@@ -205,3 +213,12 @@ def with_scope(locator)
 Then(/^I have updated the vendor home$/) do
   pending # Write code here that turns the phrase above into concrete actions
 end
+
+Then /the response is a list containing three codes/ do
+  body = JSON.parse(last_response.body)
+  c = 0
+  body['code'].each do |code|
+    c = c + 1
+  end
```

Comment:
prefer `inject` e.g.

```rb
body['code'].inject(0){|memo, _code| memo = memo + 1}
```
but also we are not even refering to `c` here, so maybe this is unused code that can be deleted?